build/sbt

```diff
@@ -20,10 +20,12 @@
 # When creating new tests for Spark SQL Hive, the HADOOP_CLASSPATH must contain the hive jars so
 # that we can run Hive to generate the golden answer.  This is not required for normal development
 # or testing.
-for i in "$HIVE_HOME"/lib/*
-do HADOOP_CLASSPATH="$HADOOP_CLASSPATH:$i"
-done
-export HADOOP_CLASSPATH
+if [ "$HIVE_HOME" ]; then
```

Comment:
I really don't know. I was thinking "If you set the variable we will try to use it" plus if isn't correct the wild card will expand to nothing anyways. The other option is to warn if `HIVE_HOME` is set but doesn't point to a directory. I'm not too convinced either way but the way it is now makes sense to me. Although the warning won't hurt.